arm/ui/settings/DriveUtils.py

```diff
@@ -192,8 +241,11 @@ def drives_update(startup=False):
     db.session.commit()
 
     # Update drive information:
-    for drive in sorted(drives_search()):  # sorted by mount point
-        app.logger.debug(drive)
+    system_drives = sorted(drives_search())
+    if len(system_drives) < 1:
+        logging.error(f"We Cant find any system drives!. {system_drives}")
```

Comment:
you've changed loggers from the `app.logger` to `logging`. consider changing this back.

Reply:
Good catch